src/odemis/acq/test/path_test.py

```diff
@@ -1352,5 +1355,95 @@ def test_guess_mode(self):
         self.assertEqual(guess, "flim-setup")
 
 
+# @skip("faster")
```

Comment:
I've seen this in quite a few places and always forgot to ask, what does it mean and why is it commented?

Reply:
It's a unittest decorator to skip the test. That's handy with Eclipse because it only allows to run all the tests on the first time you run them. It doesn't really make sense to keep it around, especially now that almost no one uses Eclipse.